src/libguac/Makefile.am

```diff
@@ -69,7 +69,9 @@ libguacinc_HEADERS =                  \
     guacamole/user.h                  \
     guacamole/user-constants.h        \
     guacamole/user-fntypes.h          \
-    guacamole/user-types.h
+    guacamole/user-types.h	      \
+    guacamole/wol.h		      \
+    guacamole/wol-constants.h
```

Comment:
I recommend aligning the `\` with the lines above.

Reply:
Ahem.  Yes.